bin/test-lxd-images

```diff
@@ -199,6 +199,8 @@ for url in $(lxc query "/1.0/instances" | jq -r .[]); do
 
     # Systemd cleanliness
     if lxc exec "${name}" -- sh -c "type systemctl" >/dev/null 2>&1; then
+        # Allow more time to boot if needed
+        lxc exec "${name}" -- systemctl --quiet is-system-running || sleep 30
```

Comment:
Similar as before, we'll normally hit this 5min after initial startup.
If the instance still isn't up, something is very wrong and it deserves a failure here :)